### DISCUSSION TITLE
最非的一集

### DISCUSSION THREAD

**Discussion by blqdswkl** [share121/inter-knot#1517] [其他]
![948F94414023B85A6EA1B08C2193416E](https://github.com/user-attachments/assets/8479081f-fe30-42a9-8216-6d28fe79da92)
![F9EC7157DABDD8E302AE77A60563BB17](https://github.com/user-attachments/assets/d1549174-6af8-465b-bc18-0104acda139a)

前面一直歪，结果199抽出了没歪，害的我白白搞这么多

**Comment by 10kaihua** [comment]
送的十连直接出ᕕ(◠ڼ◠)ᕗ

**Comment by EhuanyingK** [comment]
国际
![Image_1723173678395](https://github.com/user-attachments/assets/a7727ac2-a905-4923-bbc8-edaa7e60d160)

**Comment by Sui-Qing** [comment]
sensei绳网好玩吗？（◠ڼ◠）
